src/compiler/parser.ts

```diff
@@ -1741,6 +1777,8 @@ module ts {
                     return token === SyntaxKind.GreaterThanToken || token === SyntaxKind.OpenParenToken;
                 case ParsingContext.HeritageClauses:
                     return token === SyntaxKind.OpenBraceToken || token === SyntaxKind.CloseBraceToken;
+                case ParsingContext.JSXAttributes:
+                    return token === SyntaxKind.GreaterThanToken || token === SyntaxKind.SlashToken;

```

Comment:
Why ? close brace is an invalid token in attributes list, not a list terminator. Is there something I misunderstood ?